### PR TITLE
Language  translation 

### PR DESCRIPTION
**Pull request by razu9861**

### CODE REVIEW

File: src/app/courses/courses.component.html [hunk 1]
```diff
@@ -1,118 +1,118 @@
...
   <div class="row">
     <div class="form-group col-4">
-      <label>Course Leader</label>
+      <label i18n>Course Leader</label>
       <!-- need to create a select with options from number of members available -->
       <!-- for this to work we need all the memberes' id from the members database which is not created yet -->
       <select multiple class="form-control" formControlName="courseLeader">
         <!-- i created this according to the entry in the old bell apps couch db members database -->
-        <option *ngFor="let memb of members" value={{memb?.id}}> {{memb?.doc.firstName}} {{memb?.doc.middleName}} {{memb?.doc.lastName}}</option>
+        <option  *ngFor="let memb of members" value={{memb?.id}} i18n> {{memb?.doc.firstName}} {{memb?.doc.middleName}} {{memb?.doc.lastName}}</option>
```
Comment (lmmrssa) [inline src/app/courses/courses.component.html]: extra spacing

File: src/app/courses/courses.component.html [hunk 2]
```diff
@@ -1,118 +1,118 @@
...
   <div class="row">
     <div class="form-group col-2">
-      <label for="backgroundCol">Background Color</label>
+      <label for="backgroundCol" i18n>Background Color</label>
       <input type="color" class="form-control" formControlName="backgroundColor">
       <planet-form-error-messages [control]="courseForm.controls.backgroundColor"></planet-form-error-messages>
     </div>
     <div class="form-group col-2">
-      <label>Foreground Color</label>
+      <label> i18nForeground Color</label>
```
Comment (lmmrssa) [inline src/app/courses/courses.component.html]: it needs to be on tag

File: src/app/login/login.component.ts [hunk 3]
```diff
@@ -24,13 +24,13 @@ require('./login.scss');
           <input [(ngModel)]="model.repeatPassword" i18n-placeholder placeholder="Repeat Password" name="repeatPassword" type="password" />
         </div>
         <div class="login-actions">
-          <div><button class="ole-btn cursor-pointer">{{ createMode ? 'Create User' : 'SIGN-IN' }}</button></div>
+          <div><button class="ole-btn cursor-pointer" i18n>{{ createMode ? 'Create User' : 'SIGN-IN' }}</button></div>
```
Comment (lmmrssa) [inline src/app/login/login.component.ts]: this probably will not work

File: src/app/login/login.component.ts [hunk 4]
```diff
@@ -24,13 +24,13 @@ require('./login.scss');
...
         <div class="login-actions">
-          <div><button class="ole-btn cursor-pointer">{{ createMode ? 'Create User' : 'SIGN-IN' }}</button></div>
+          <div><button class="ole-btn cursor-pointer" i18n>{{ createMode ? 'Create User' : 'SIGN-IN' }}</button></div>
           <a [routerLink]="createMode ? ['/login'] : ['newuser']">
             {{ createMode ? 'Already have an account?' : 'Are you new?' }}
           </a>
         </div>
       </form>
-      <div id="login-status">{{message}}</div>
+      <div id="login-status" i18n>{{message}}</div>
```
Comment (lmmrssa) [inline src/app/login/login.component.ts]: this will not work

File: src/app/resources/resources.component.html [hunk 5]
```diff
@@ -1,21 +1,21 @@
 <form (ngSubmit)="submitResource()" #resourcesForm="ngForm">
   <div class="form-group">
-    <label>File Type:</label>
+    <label i18n >File Type:</label>
```
Comment (lmmrssa) [inline src/app/resources/resources.component.html]: extra space

File: src/app/resources/resources.component.html [hunk 6]
```diff
@@ -1,21 +1,21 @@
...
-    <label>File Upload:</label>
+    <label i18n>File Upload:</label>
     <input type="file" (change)="bindFile($event)" />
   </div>
-  <button type="submit">Submit</button>
+  <button i18n type="submit">Submit</button>
 </form>
 <div>
-  <span>{{message}}</span>
+  <span i18n>{{message}}</span>
```
Comment (lmmrssa) [inline src/app/resources/resources.component.html]: it might not work here

**Comment by mappuji** [comment]
Do review with this step please 
https://www.youtube.com/watch?v=SmZgAULwoh4

File: src/app/meetups/meetups-add.component.ts [hunk 7]
```diff
@@ -5,16 +5,16 @@ import { CouchService } from '../shared/couchdb.service';
   template: `
   <p>{{message}}</p>
   <form class="form-horizontal" (ngSubmit)="onSubmit(meetupForm.value)" #meetupForm="ngForm">
-    <legend>Start a New Meetup</legend>
+    <legend i18n>Start a New Meetup</legend>
     <div class="form-group col-md-4">
       <label class="control-label">Title</label>
```
Comment (nirojdyola) [inline src/app/meetups/meetups-add.component.ts]: no translation here

**Comment by lmmrssa** [comment]
It has been old and we are implementing it on new PR anyway

**Pull request closed without merging**